src/EditorFeatures/Core/Implementation/Diagnostics/AbstractDiagnosticsTaggerProvider.cs

```diff
@@ -152,28 +152,28 @@ private void ProduceTags(TaggerContext<TTag> context, DocumentSnapshotSpan spanT
             var suppressedDiagnosticsSpans = (NormalizedSnapshotSpanCollection?)null;
             buffer?.Properties.TryGetProperty(PredefinedPreviewTaggerKeys.SuppressDiagnosticsSpansKey, out suppressedDiagnosticsSpans);
 
-            var eventArgs = _diagnosticService.GetDiagnosticsUpdatedEventArgs(
+            var buckets = _diagnosticService.GetDiagnosticBuckets(
                 workspace, document.Project.Id, document.Id, context.CancellationToken);
```

Comment:
this was always super oddly named.  nothing is being named.  nothing is an event.  nothing is an arg.  This was very confusing when doing the LSP changes.  'bucket' seems like a more reasonable name in that given any Doc in the IDIagnosticsService, there are N buckets that have diagnostics associated with them that are aggregated to form the full set of diagnostics for that document.